specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json

```diff
@@ -271,6 +271,12 @@
           "application/json"
         ],
         "responses": {
+          "200": {
+            "description": "Export job status, project metadata, and assets.",
+            "schema": {
+              "$ref": "#/definitions/ExportJobState"
+            }
+          },
```

Comment:
*Copied from #17406 - not sure why there's two PRs for the same change, but in different operations.*

I'm confused. An LRO could return either a 200 with the completed state, or a location for updating an LRO object state? What would a method look like in this case for statically-typed languages like C# and Java? How would customers know what they will get, or is the idea that an operation object e.g., `Operation<JobState>` in .NET, will just come back in a completed state?

/cc @johanste

Reply:
I had some followup, but see my comments in #17406 for updates.